ambari-server/src/main/java/org/apache/ambari/server/configuration/Configuration.java

```diff
@@ -4138,6 +4146,20 @@ public String getPasswordPolicyDescription() {
     return getProperty(PASSWORD_POLICY_DESCRIPTION);
   }
 
+  /**
+   * @return Password policy history count
+   */
+  public int getPasswordPolicyHistoryCount() {
+    int historyCount = Integer.parseInt(getProperty(PASSWORD_POLICY_HISTORY_COUNT));
+    if(historyCount < 1){
+      historyCount = 1;
+    }
+    if(historyCount > 10){
+      historyCount = 10;
```

Comment:
It would be best to add some logging here to facilitate debugging, especially in cases where the value set by the user is greater than 10 but is not effective. Without logging, it can be quite confusing. Additionally, describe in the documentation that the maximum number of times is 10 for performance considerations.

Reply:
Thank you @JiaLiangC , I have added some logging for the limits and updated the description.